src/com/geekhub/hw3/user/User.java

```diff
@@ -0,0 +1,48 @@
+package com.geekhub.hw3.user;
+
+public class User implements Comparable {
+
+    private String login;
+    private String password;
+
+    public User(String login, String password) {
+        this.login = login;
+        this.password = password;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    /**
+     * Compares users login and password
+     * @param o User to compare
+     * @return User comparison result status: 0 - equals, 1 - login mismatch, 2 - password mismatch
+     */
+    @Override
+    public int compareTo(Object o) {
```

Comment:
Comparable interface and compareTo method used for sorting arrays, collections etc. 
Read Comparable::compareTo JavaDoc to know what sould return this method. 
In case if your method return 0, 1 or 2 sorting will work incorrectly.
Checking login and password logic should be in UserManager class not in User.